### PR TITLE
aplicando mvp

### PR DESCRIPTION
**Pull request by dalgarins**
ejemplo de implementacion de clean architecture y mvp sobre arquitectura offline

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```
Comment (droidpl) [inline .idea/compiler.xml:2]: Project settings should not be submitted

File: .idea/copyright/profiles_settings.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<component name="CopyrightManager">
```
Comment (droidpl) [inline .idea/copyright/profiles_settings.xml:1]: Project items should not be submitted

File: demo-sdk/build.gradle [hunk 3]
```diff
@@ -24,7 +24,7 @@ android {
 }
```
Comment (droidpl) [inline demo-sdk/build.gradle:24]: Latest version should remain

**Review by droidpl — COMMENTED** [review]
Nice implementation, but I would keep it in a separated brach just in case someone wants to check it out. What do you think?

**Comment by dalgarins** [comment]
Of course, leave this branch apart, everyone who sees the video wants to see the original code, this branch is going to be a guide on how to implement it with clean.

**Comment by dalgarins** [comment]
Implementation was improved to run a new merge.

**Pull request closed without merging**